src/connectors/index.ts

```diff
@@ -32,8 +33,7 @@ export const walletConnectXDAI = new WalletConnectConnector({
     100: 'https://poa-xdai.gateway.pokt.network/v1/lb/61140fc659501900341babff'
   },
   bridge: 'https://walletconnect-relay.minerva.digital',
-  qrcode: true,
-  pollingInterval: 15000
```

Comment:
Some reason to we remove the pollingInterval?

Reply:
Now pollingInterval is not a valid argument for `WalletConnectConnectorArguments`, I had to delete that argument when updated we3-react library to make Uauth works well. Thanks for the review and recommendations!

Reply:
Nice